artemis-protocols/artemis-openwire-protocol/src/test/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireMessageConverterTest.java

```diff
@@ -135,6 +138,15 @@ public void testProperties() throws Exception {
       }
    }
 
+   @Test
+   public void testEmptyMapMessage() throws Exception {
+      CoreMessage artemisMessage = (CoreMessage) OpenWireMessageConverter.inbound(new ActiveMQMapMessage().getMessage(), openWireFormat, null);
+      ActiveMQBuffer buffer = artemisMessage.getDataBuffer();
+      TypedProperties map = new TypedProperties();
+      buffer.resetReaderIndex();
+      map.decode(buffer.byteBuf());
```

Comment:
Might be nice to check the CoreMessage actually represents a MapMessage, and maybe what comes out of the decode matches expectations of what went in, i.e verify some behaviours of the conversion and not just simply that it didnt throw. Essentially most of what the systests is really doing around verifying the area/behaviour changed, but much faster.

Reply:
Done.